package.json

```diff
@@ -10,15 +10,15 @@
     "tsc": "lerna run tsc",
     "link": "lerna link",
     "version": "lerna version",
-    "lint": "lerna run --parallel lint",
+    "lint": "lerna run --no-bail --parallel lint",
```

Comment:
What "--no-bail" is necessary? To show all errors, not just first ones?

Reply:
Yes, especially considering that tests from the remaining packages will still continue to run, we just won't see their results (and will only be able to detect whether they are still running by checking whether there are some `node` processes)